xbmc/interfaces/json-rpc/VideoLibrary.cpp

```diff
@@ -1123,8 +1123,25 @@ void CVideoLibrary::UpdateVideoTag(const CVariant &parameterObject, CVideoInfoTa
 
   if (ParameterNotNull(parameterObject, "track"))
     details.m_iTrack = (int)parameterObject["track"].asInteger();
-  if (ParameterNotNull(parameterObject, "rating"))
```

Comment:
Can we keep `rating` and `votes` support here which will simply modify the default rating/votes? That way the API would stay backwards compatible.